moto/sagemaker/responses.py

```diff
@@ -274,3 +274,29 @@ def delete_notebook_instance_lifecycle_config(self):
             )
         )
         return 200, {}, json.dumps("{}")
+
+    @amzn_request_id
+    def list_training_jobs(self):
+        next_token = self._get_param("NextToken")
+        max_results = self._get_int_param("MaxResults")
+        creation_time_after = self._get_param("CreationTimeAfter")
+        creation_time_before = self._get_param("CreationTimeBefore")
+        last_modified_time_after = self._get_param("LastModifiedTimeAfter")
+        last_modified_time_before = self._get_param("LastModifiedTimeBefore")
+        name_contains = self._get_param("NameContains")
+        status_equals = self._get_param("StatusEquals")
+        sort_by = self._get_param("SortBy", "CreationTime")
+        sort_order = self._get_param("SortOrder", "Ascending")
+        response = self.sagemaker_backend.list_training_jobs(
```

Comment:
This should be done in a `try-except`, to ensure we properly format any errors that it might throw.
See `create_notebook_instance_lifecycle_config` above for an example